bestPractices.js

```diff
@@ -0,0 +1,333 @@
+const loadElements = () => {
+  let taskArray = [];
+  taskArray = retrieveData();
+  taskArray
+    .sort((x, y) => x.index - y.index)
+    .forEach((todo) => {
+      const taskElement = createDisplayElement(todo);
+      appendToDOM(taskElement);
+    });
+};
+
+const loadRefreshIcon = () => {
+  const refreshIcon = document.getElementById('list_refresh');
+  refreshIcon.src = Refresh;
+  refreshIcon.alt = 'refresh';
+  refreshIcon.setAttribute('class', 'header-icon');
+  refreshIcon.addEventListener('click', () => {
+    window.location.reload();
+  });
+};
+
+const loadEnterIcon = () => {
+  const enterIcon = document.getElementById('enter_newtodo');
+  enterIcon.src = Enter;
+  enterIcon.alt = 'enter';
+  enterIcon.setAttribute('class', 'icon');
+  // add eventlistner for enter key
+  enterIcon.addEventListener('click', () => {
+    // if todo is written or add to your list field is not empity
+    if (addTodoForm.elements['add-task'].value) {
+      const todosArray = retrieveData();
+      const newTodo = new Todo(
+        addTodoForm.elements['add-task'].value,
+        false,
+        todosArray.length + 1,
+      );
+      Todo.addTodo(newTodo);
+      const todoElement = createDisplayElement(newTodo);
+      appendToDOM(todoElement);
+      addTodoForm.reset();
+    } else if (document.getElementsByClassName('edit-todo-input')[0]) { // if add todo is empity and if someone is yousing phone and want to edit a task
+      updateTodoHandler();
+    }
+  });
+};
+
+const createCheckbox = (status) => {
+  const checkbox = document.createElement('input');
+  checkbox.setAttribute('class', 'checkbox');
+  checkbox.setAttribute('type', 'checkbox');
+  if (status) {
+    checkbox.checked = true;
+  }
+  checkbox.addEventListener('change', checkboxStatus);
+  return checkbox;
+};
+
+const addTodoForm = document.getElementById('todo-form');
```

Comment:
- According to Javascript Best Practices would be great if you declare variables before using it. I notice you are using this Variable before declaring it in previous functions (lines, 30, 33 40 are some examples ).